FAST-LIO/config/mid360.yaml

```diff
@@ -12,15 +13,35 @@ mapping:
     acc_cov: 0.1
     gyr_cov: 0.1
     b_acc_cov: 0.0001
-    b_gyr_cov: 0.0001
+    b_gyr_cov: 0.0001               
     fov_degree:    180
     det_range:     260.0
     extrinsic_T: [ -0.011, -0.02329, 0.04412 ] # Lidar center w/r/t IMU
     extrinsic_R: [ 1, 0, 0,
                    0, 1, 0,
                    0, 0, 1]
+    extrinT_Gnss2Lidar: [ 0, 0, 0]
+    extrinR_Gnss2Lidar: [ 1, 0, 0, 
+                          0, 1, 0, 
+                          0, 0, 1]
 
 publish:     
     scan_publish_en:  true       # 'false' will close all the point cloud output
     dense_publish_en: true       # false will low down the points number in a global-frame point clouds scan.
-    scan_bodyframe_pub_en: true  # output the point cloud scans in IMU-body-frame
\ No newline at end of file
+    scan_bodyframe_pub_en: true  # output the point cloud scans in IMU-body-frame
+    gps_publish_en: true         
+
+# Loop closure
+    loopClosureEnableFlag: true
+    loopClosureFrequency: 4.0                     # Hz, regulate loop closure constraint add frequency
+    surroundingKeyframeSize: 50                   # submap size (when loop closure enabled)
+    historyKeyframeSearchRadius: 20.0            # meters, key frame that is within n meters from current pose will be considerd for loop closure
+    historyKeyframeSearchTimeDiff: 30.0           # seconds, key frame that is n seconds older will be considered for loop closure
+    historyKeyframeSearchNum: 25                  # number of hostory key frames will be fused into a submap for loop closure
+    historyKeyframeFitnessScore: 0.3              # icp threshold, the smaller the better alignment
+
+# GPS Settings
+    useImuHeadingInitialization: true           # if using GPS data, set to "true"
+    useGpsElevation: false                      # if GPS elevation is bad, set to "false"
+    gpsCovThreshold: 2.0                        # m^2, threshold for using GPS data
+    poseCovThreshold: 0 #25.0                      # m^2, threshold for using GPS data  pose covariance threshold from isam2
```

Comment:
let's replace "threshold" with "minimum" or "maximum" as appropriate for improved readability